lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -152,11 +172,33 @@ def get(self, request, *args, **kwargs):
         transformers = BlockStructureTransformers()
         transformers += get_course_block_access_transformers(request.user)
         transformers += [
-            BlocksAPITransformer(None, None, depth=3),
+            BlocksAPITransformer(None, None, depth=4),
         ]
 
         course_blocks = get_course_blocks(request.user, course_usage_key, transformers, include_completion=True)
 
+        has_visited_course = False
+        resume_course_url = None
+        for block_key in course_blocks:
+            if course_blocks.get_xblock_field(block_key, 'category') == 'course':
```

Comment:
So call me crazy, but I _think_ there might be an easier way to get the course block. You might be able to use `course_blocks.root_block_usage_key` although that might only give the key which I think we already have... Or maybe you can do something like `course_blocks[course_key]` to get the course block? If not, look into `topological_traversal` which should begin at the course level so you can just take the first element there.